docs/src/pages/components/icons/CreateSvgIcon.tsx

```diff
@@ -1,29 +1,23 @@
 import * as React from 'react';
-import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import { createSvgIcon } from '@material-ui/core/utils';
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      '& > svg': {
-        margin: theme.spacing(2),
-      },
-    },
-  }),
-);
-
 const HomeIcon = createSvgIcon(
   <path d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z" />,
   'Home',
 );
 
 export default function CreateSvgIcon() {
-  const classes = useStyles();
-
   return (
-    <div className={classes.root}>
+    <Box
+      sx={{
+        '& > svg': {
```

Comment:
We can make it generic. Easier to move code around:

```suggestion
        '& > :not(style)': {
```

x the other cases

Reply:
Resolved